wsd/DocumentBroker.cpp

```diff
@@ -878,6 +881,20 @@ bool DocumentBroker::download(
     {
         LOG_DBG("CheckFileInfo for docKey [" << _docKey << ']');
         std::chrono::steady_clock::time_point start = std::chrono::steady_clock::now();
+        if (!wopiFileInfo)
+        {
+            auto poller = std::make_shared<TerminatingPoll>("CFISynReqPoll");
+            poller->startThread();
+            CheckFileInfo checkFileInfo(poller, session->getPublicUri(), [](CheckFileInfo&) {});
```

Comment:
This dereferences "session", but later on on line 903 there is a check of session against null., which suggests it could be null here too?